Hoxtygen/next-fullstack
# Feat :implement password reset

## What does this PR do?
Implements password reset functionalities

## Description of Task to be completed?
* Create password reset request components
* Create password reset components
* Integrate API

## How should this be manually tested?
* Clone the repository.
* On your PC open a terminal and run git clone <repo-url>.
* `cd` into the project folder and run`yarn install`.
* Run `yarn dev` to start the application
* Click on the login link, and on login page click on `reset password` link.
* Enter your registered email to receive password reset. link.
* Check your mail and click on the link provided.
* Follow the link and reset your password.

## What are the relevant Jira board stories?
[HOM-4](https://wasiu-dev.atlassian.net/jira/software/projects/HOM/boards/2?selectedIssue=HOM-4)
[HOM-53](https://wasiu-dev.atlassian.net/jira/software/projects/HOM/boards/2?selectedIssue=HOM-53)

[HOM-4]: https://wasiu-dev.atlassian.net/browse/HOM-4?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[HOM-53]: https://wasiu-dev.atlassian.net/browse/HOM-53?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
